BackEnd/routes/members.py

```
@@ -1,15 +1,101 @@
+    try:
+        members = []
+        async for member in members_collection.find({}):
+            members.append({
+                "id": str(member["_id"]),
+                "name": member.get("name", ""),
+                "points": member.get("points", 0),
+                "level": member.get("level", "Bronze"),
+                "badges": member.get("badges", [])
+            })
+        return {"status": "success", "data": {"members": members, "total": len(members)}}
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)